src/OrbitVulkanLayer/VulkanLayerController.h

```diff
@@ -87,11 +87,36 @@ class VulkanLayerController {
     // Advance linkage for next layer.
     layer_create_info->u.pLayerInfo = layer_create_info->u.pLayerInfo->pNext;
 
+    // Ensure that the extensions that the layer uses are requested in the CreateInstance call.
+    // As we can and should not modify the given create_info, we need to create a modified version
+    // containing the required extension. So first create a copy of the given extension names.
+    std::vector<std::string> all_extension_names{};
+    all_extension_names.assign(
+        create_info->ppEnabledExtensionNames,
+        create_info->ppEnabledExtensionNames + create_info->enabledExtensionCount);
+
+    // To the extensions, requested by the game, add our required extension (if not already
+    // present).
```

Comment:
I would revert the sentence, like "Add our required extension (if not already present), to the extensions requested by the game."